modules/govuk_mysql/templates/usr/local/bin/xtrabackup_s3_restore.erb

```diff
@@ -21,8 +22,13 @@ else
   govuk_puppet --disable "MySQL environment sync (by $0)"
 
   # Create directories to restore into
-  for dir in "$BASE_DIR" "$INCR_DIR"; do
-    mkdir -p $dir
+  if [[ ! -d "CURRENT_RESTORE" && ! -d "$BASE_DIR" && ! -d "$INCR_DIR" ]]; then
+    for dir in "$BASE_DIR" "$INCR_DIR"; do
+    mkdir $dir
+    done
+  else
+    for dir in "$BASE_DIR" "$INCR_DIR"; do
+    rm -rf $dir/*
```

Comment:
We think it would be more clear to replace all of the loops and conditionals above with just these 2 commands. One to delete the directories no matter whether they exist or not and one to create them empty.

```bash
rm -rf $BASE_DIR $INCR_DIR
mkdir -p $BASE_DIR $INCR_DIR
```

Ana pointed out that if `$BASE_DIR` or `$INCR_DIR` in set to an empty string this change would remove the root directory.